ui/lib/pki/addon/components/pki-issuer-cross-sign.hbs

```diff
@@ -0,0 +1,91 @@
+<h3 for="intermediates to cross-sign" class="title is-4">Intermediates</h3>
+<div class="box is-fullwidth has-only-top-shadow">
+  <p class="has-bottom-margin-xl">
+    Intermediates are identified by specifying the mount path and the issuer’s current name, as well as the new name it will
+    have when it is imported to Vault.
+  </p>
+  {{#if this.signedIssuers}}
+    <div class="is-flex-start">
+      {{#each this.inputFields as |column|}}
+        <div class="is-flex-1 basis-0 has-text-grey has-bottom-margin-xs">
+          <h2 data-test-info-table-label={{column.key}}>
+            {{column.label}}
+            {{#if column.helpText}}
+              <InfoTooltip @verticalPosition="above" @horizontalPosition="center">{{column.helpText}}</InfoTooltip>
+            {{/if}}
+          </h2>
+        </div>
+      {{/each}}
+    </div>
+    <div class="box is-fullwidth has-only-top-shadow">
+      {{#each this.signedIssuers as |crossSignRow|}}
+        <div class="box is-marginless no-top-shadow has-slim-padding">
```

Comment:
I considered making this table a component `<InfoTableColumns>`, but there aren't any current design plans to use this table display display elsewhere. I can pull out into a component, though, if preferred!

<img width="1364" alt="Screen Shot 2023-01-24 at 5 23 25 PM" src="https://user-images.githubusercontent.com/68122737/214459171-a39a6feb-23d4-4d9e-9115-a7f9543336fc.png">

Reply:
This component isn't terribly large and if we're unsure if the pattern will reappear I would suggest leaving as is and we can always come back and rework it into a generic component if need be.